__tests__/implementation.test.js

```diff
@@ -92,4 +92,25 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles", () => {
+    it("should respond with status 200 and an array of articles", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.articles).toBeInstanceOf(Array);
+          response.body.articles.forEach((article) => {
```

Comment:
please add a test for the length of the array before you test multiple things using an array. If you receive an empty array the tests inside the forEach would be skipped but the length test would still run